css/menu-block.css

```diff
@@ -3,25 +3,28 @@
  * book navigation block in unl_fourone_preprocess_block().
  */
 
-.block-menu-block a {color: #333;}
-.block-menu-block a:hover {color: #d00000;}
-.block-menu-block a.active {background-color: #F2E8B1 !important;}
-.block-menu-block h3 {background-color: #454542; color: #FFF; font-size: inherit; text-transform: uppercase; margin: 0; padding: 1em;}
-.block-menu-block h3 a {color: #fff; border: none;}
+.block-menu-block a {padding-right: 1.777em; color: #474746;}
+.block-menu-block a:hover {cursor: pointer; color: #d00000;}
+.block-menu-block a.active {color: #6a0000;}
+.block-menu-block h3 {background-color: #474746; color: #fefdfa; font-size: .802rem; text-transform: uppercase; margin: 0; line-height: 1.333}
+.block-menu-block h3 a {display: block; padding-top: 1.333em; padding-bottom: 1.069em; padding-left: 1em; border: none; color: #fefdfa;}
```

Comment:
Move padding to the h3 for the use case where the header isn't linked (an option in Menu Block)